help/resize-the-compose-box.md

```diff
@@ -10,17 +10,18 @@ box also stretches automatically when you type a long message.
 
 {!start-composing.md!}
 
-1. Click on the **<i class="fa fa-chevron-up"></i> icon** in the top
-   right corner of the compose box.
+1. Click on the **<i class="zulip-icon zulip-icon-expand-diagonal"></i> icon** in the top
+   right corner of the compose box, which is visible on hovering over the message content area.
```

Comment:
I looked through the help center articles but it doesn't seem we have a dedicated term for the textarea / preview, and we just call it the compose box, same as the overall area. Since it needed to be distinguished here to specify where exactly to hover, I call it "message content area"... @alya Let me know if something else would be better!